### PR TITLE
General fixes for Fixes

### PR DESCRIPTION
**Pull request by dcrissman**
Changes made by @derek63 and myself, primarily to address changes to the migrator entities.

### CODE REVIEW

File: consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/MigrationJob.java [hunk 1]
```diff
@@ -18,382 +18,393 @@
...
-	protected Map<String, JsonNode> findDestinationData(LightblueRequest findRequest) throws IOException {
-		return getJsonNodeMap(getDestinationClient().data(findRequest, JsonNode[].class), getJobConfiguration().getDestinationEntityKeyFields());
-	}
-
-	protected LinkedHashMap<String, JsonNode> getJsonNodeMap(JsonNode[] results, List<String> entityKeyFields) {
-		LinkedHashMap<String, JsonNode> resultsMap = new LinkedHashMap<>();
-		for(JsonNode result : results) {
-			StringBuilder resultKey = new StringBuilder();
-			for(String keyField : entityKeyFields) {
-				resultKey.append(result.findValue(keyField)).append("|||");
```
Comment (jewzaam) [inline consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/MigrationJob.java]: The delimiter here was to ensure value fields don't incorrectly collide.  For example, if you have numeric identify fields it's very possible for identity field 123456 to collide with an object that has two identity fields: 123 and 456.

**Comment by jewzaam** [comment]
Review complete, just the one comment about the delimiter for the map key.

File: consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/MigrationJob.java (hunk 1 above)
Comment (dcrissman) [inline consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/MigrationJob.java]: @derek63 would you mind taking a look at this?

Comment (jewzaam) [inline consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/MigrationJob.java]: @dcrissman this is your PR.  Would you rather we merge this PR and deal with the delimiter in another PR?

Comment (derek63) [inline consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/MigrationJob.java]: Delimiter makes sense for the multiple-field identities, just need to add logic to parse/remove it  when building the query to lightblue based on identity fields (as we were submitting queries with it in there)

Comment (dcrissman) [inline consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/MigrationJob.java]: I can no longer reproduce so I  reverted that particular commit. If it comes up again, we can tackle it in another issue.

**Pull request merged**